Formula/nginx.rb

```diff
@@ -143,7 +143,11 @@ def caveats
   end
 
   service do
-    run [opt_bin/"nginx", "-g", "'daemon off;'"]
+    if OS.mac?
+      run [opt_bin/"nginx", "-g", "daemon off;"]
+    else
+      run [opt_bin/"nginx", "-g", "'daemon off;'"]
```

Comment:
Service blocks should probably take care of proper quoting on Linux. Thoughts, @SMillerDev?

Reply:
@carlocab @SMillerDev agreed 👍🏻